test/e2e/util/util.go

```diff
@@ -1506,6 +1508,282 @@ func EnsureGuestWebhooksValidated(t *testing.T, ctx context.Context, guestClient
 	})
 }
 
```

Comment:
where is EnsureGlobalPullSecret invoked?

Reply:
I probably override the lie from a rebase.